htdocs/takepos/admin/setup.php

```diff
@@ -303,6 +304,12 @@
 print $form->selectarray('TAKEPOS_SORTPRODUCTFIELD', $array, (empty($conf->global->TAKEPOS_SORTPRODUCTFIELD) ? 'rowid' : $conf->global->TAKEPOS_SORTPRODUCTFIELD), 0, 0, 0, '', 1);
 print "</td></tr>\n";
 
+print '<tr class="oddeven"><td>';
+print $langs->trans('TakeposGroupSameProduct');
+print '<td colspan="2">';
+print ajax_constantonoff("TAKEPOS_GROUP_SAME_PRODUCT", array(), $conf->entity, 0, 0, 1, 0);
```

Comment:
If you use the new feature "forcereload" (param 1 at position 6), it means clicking on image is enough to switch.
This also means you don't have to make the 
"$res = dolibarr_set_const($db, 'TAKEPOS_GROUP_SAME_PRODUCT', ..."
when you click on save button or you will erase the setup

Reply:
@eldy 
Thanks for your remark, I deleted this line.